src/pkg/cluster/zarf.go

```diff
@@ -33,11 +33,11 @@ func (c *Cluster) GetDeployedZarfPackages(ctx context.Context) ([]types.Deployed
 	listOpts := metav1.ListOptions{LabelSelector: ZarfPackageInfoLabel}
 	secrets, err := c.Clientset.CoreV1().Secrets(ZarfNamespaceName).List(ctx, listOpts)
 	if err != nil {
-		return nil, err
+		return []types.DeployedPackage{}, err
 	}
 
-	errs := []error{}
-	deployedPackages := []types.DeployedPackage{}
+	errs := make([]error, 0)
+	deployedPackages := make([]types.DeployedPackage, 0)
```

Comment:
I think it is fine with the original implementation. I am generally not a fan of make to create a struct or slice unless it is for the explicit reason to pre allocate capacity.

Reply:
I can get behind this as long as we're consistent. I've learned to allocate with make and goland was flagging it, but afaik it compiles exactly the same.